### PR TITLE
Rainmeter.Beta version 4.4.0.3506

### PR DESCRIPTION
**Pull request by brianferguson**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22394)

### CODE REVIEW

File: manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+PackageIdentifier: Rainmeter.Rainmeter.Beta
+PackageVersion: 4.4.0.3506
+PackageLocale: en-US
+Publisher: Rainmeter
+PublisherUrl: https://www.rainmeter.net
+PublisherSupportUrl: https://forum.rainmeter.net
+#PrivacyUrl: 
+Author: Rainmeter Team
+PackageName: Rainmeter.Beta
```
Comment (ItzLevvie) [inline manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.locale.en-US.yaml:12]:
```suggestion
PackageName: Rainmeter (Beta)
```

File: manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Rainmeter.Rainmeter.Beta
+PackageVersion: 4.4.0.3506
```
Comment (ItzLevvie) [inline manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.4 beta r3506
```

File: manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Rainmeter.Rainmeter.Beta
+PackageVersion: 4.4.0.3506
```
Comment (ItzLevvie) [inline manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.installer.yaml:5]:
```suggestion
PackageVersion: 4.4 beta r3506
```

File: manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Rainmeter.Rainmeter.Beta
+PackageVersion: 4.4.0.3506
```
Comment (ItzLevvie) [inline manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.yaml:5]:
```suggestion
PackageVersion: 4.4 beta r3506
```

File: manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+Author: Rainmeter Team
+PackageName: Rainmeter.Beta
+PackageUrl: https://www.rainmeter.net
+License: GPLv2
+LicenseUrl: https://github.com/rainmeter/rainmeter/blob/master/LICENSE
+Copyright: © 2021 Rainmeter Team
+CopyrightUrl: https://github.com/rainmeter/rainmeter/blob/master/LICENSE
+ShortDescription: Desktop Customization Tool
+Description: Rainmeter allows you to display customizable skins on your desktop, from hardware usage meters to fully functional audio visualizers. You are only limited by your imagination and creativity.
+Moniker: rainmeter
```
Comment (ItzLevvie) [inline manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.locale.en-US.yaml:20]:
```suggestion
Moniker: rainmeter-beta
```

File: manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+#  Scope:
```
Comment (ItzLevvie) [inline manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.installer.yaml:14]:
```suggestion
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
image
Reasons for change:
* `PackageVersion:` needs to match **A**nd or **R**emove **P**rograms. (Marketing version can be used in later ManifestVersions)
* `Scope:` This depends if the application requires Administrator to be installed
* `PackageName:` This also needs to match ARP, however since the Name is just "Rainmeter" its better to display "Rainmeter Beta"
* `Moniker:` rainmeter will cause issues with the stable version, and now you can install it via the PackageName you provided

Request to you as a developer?
* Change the `Version` to `4.4.0.3506`
* Change the `Name` to `Rainmeter Beta`

And are you guys finally adding support for Silent install since its in this beta?

Edit: Took too long to type so ItzLevvie did the changes 😁

File: manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.locale.en-US.yaml (hunk 1 above)
Comment (brianferguson) [reply manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.locale.en-US.yaml:12]: Isn't using the `.Beta` naming conventions recommended?
Google does it for [Chrome](https://github.com/microsoft/winget-pkgs/blob/master/manifests/g/Google/Chrome/Beta/92.0.4515.81/Google.Chrome.Beta.installer.yaml#L2).

File: manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.locale.en-US.yaml (hunk 2 above)
Comment (brianferguson) [reply manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.locale.en-US.yaml:5]: I think leaving a full version on this line is better.

File: manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.locale.en-US.yaml (hunk 1 above)
Comment (OfficialEsco) [reply manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.locale.en-US.yaml:12]: Not in the PackageName
https://github.com/microsoft/winget-pkgs/blob/master/manifests/g/Google/Chrome/Beta/92.0.4515.81/Google.Chrome.Beta.locale.en-US.yaml#L10

Comment (brianferguson) [reply manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.locale.en-US.yaml:12]: Ah, I see. Thanks!

File: manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.locale.en-US.yaml (hunk 2 above)
Comment (OfficialEsco) [reply manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3506/Rainmeter.Rainmeter.Beta.locale.en-US.yaml:5]: This is the version you guys are displaying in the Control Panel, which is why it has to match that.

**Comment by brianferguson** [comment]
> 
> 
> image
> Reasons for change:
> 
>     * `PackageVersion:` needs to match **A**nd or **R**emove **P**rograms. (Marketing version can be used in later ManifestVersions)
> 
>     * `Scope:` This depends if the application requires Administrator to be installed
> 
>     * `PackageName:` This also needs to match ARP, however since the Name is just "Rainmeter" its better to display "Rainmeter Beta"
> 
>     * `Moniker:` rainmeter will cause issues with the stable version, and now you can install it via the PackageName you provided
> 
> 
> Request to you as a developer?
> 
>     * Change the `Version` to `4.4.0.3506`
> 
>     * Change the `Name` to `Rainmeter Beta`
> 
> 
> And are you guys finally adding support for Silent install since its in this beta?
> 
> Edit: Took too long to type so ItzLevvie did the changes 😁

Yes, we added silent support since 3502, but version 3506 has all the bugs worked out.

Hmmm, I didn't realize that these manifest fields needed to exactly match what our installer uses. I will talk this over with the team an get it remedied soon.

However, since we will be changing these fields, we will put out a new beta and this manifest can be closed. We will re-submit once the new beta is out.

**Pull request closed without merging**